prow/cmd/mkpod/local-kind.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+job=${1:-pull-test-infra-yamllint}
```

Comment:
Quote your var expansions

Reply:
Will do, I'm a bash noob and don't see how that would make a difference here though. Is the concern that without quotes `job` could be a list if there are spaces?

Reply:
Yes. Always quote them so that the absence of quotes is an intentional choice and can be interpreted as such.